src/github.com/getlantern/errlog/errlog.go

```diff
@@ -0,0 +1,529 @@
+/*
+Package errlog defines error types used across Lantern project and implements
+functions to manipulate them.
+
+  var elog = errlog.ErrorLoggerFor("package-name")
+  if n, err := Foo(); err != nil {
+    elog.Log(err, errlog.WithOp("foo"), errlog.WithField("foo": "bar"))
+  }
+
```

Comment:
Why not just use the struct directly, like this:

```golang
elog.Log(&Error{
    Err: err,
    Op: "foo",
    Extras: map[string]string {
        "foo": "bar",
    },
})
```

To me, that seems more readable and also more conventional.